arduino-ide-extension/src/browser/serial/monitor/serial-monitor-send-input.tsx

```diff
@@ -6,6 +6,74 @@ import { BoardsServiceProvider } from '../../boards/boards-service-provider';
 import { MonitorModel } from '../../monitor-model';
 import { Unknown } from '../../../common/nls';
 
+class RingList {
+  private ring: string[];
+  private size: number;
+  private begin: number;
+  private index: number;
+  private end: number;
+
+  constructor(size: number = 100) {
+    this.init = this.init.bind(this);
+    this.push = this.push.bind(this);
+    this.prev = this.prev.bind(this);
+    this.next = this.next.bind(this);
+    this.init(size);
+  }
+
+  private init(size: number = 100)
```

Comment:
Should be `private init(size = 100) {`

Please respect the position of the curly braces.